2-0-data-structures-and-algorithms/src/test/java/com/bobocode/array_list/ArrayListTest.java

```diff
@@ -287,7 +375,11 @@ public void removeLastElementByIndex() {
     @Test
     @Order(27)
     public void removeElementByIndexOutOfBounds() {
```

Comment:
@erneman1 please rename to `removeElementByIndexThrowsExceptionWhenIndexIsOutOfBounds`